### ISSUE TITLE
Wiredep doesn't fetch dependencies via SVN

### ISSUE THREAD

**Issue by Jezzuz** [stephenplusplus/grunt-wiredep#129]
I have a Module A with a lot of dependencies.

I have a Module B dependant on A. bower fetches A into B's library via 
"dependencies": {
    "ModuleA": "svn+http://path/to/module/A"
  }
bower goes all well. A's dependencies and A get installed into B.

Somehow wiredep doesn't work tho. Neither A or it's dependencies are wired into B. Utiliting the "onMainNotFound"-Option I see that no main has been found for Module A. Module A has its Main clearly declared in its bower.json. 

Am I missing something, or does wiredep simply not work with svn paths?

**Comment by stephenplusplus** [comment]
Can you put a sample repo together with the directory structure and bower.json's you ended up with? I haven't used Bower with SVN, but I can't immediately think of how wiredep's code would care where the package files came from, as long as they're there.

**Comment by Jezzuz** [comment]
Hello Stephen and thank you for your quick reply.

I was able to get rid of the misbehaviour myself: In Module B ( which is dependant on Module A, see above ) I started off with using bower without `.bowerrc`, which installed deps into `bower_components`. Later I included `.bowerrc` to have deps go into a folder `lib`. I just found out, I never deleted `bower_components`. 

Module A was existant in `bower_components` in a state where its "main" was not yet declared correctly. Upon deleting `bower_components` everything worked instantly. 

The only question remaining is, shouldn't wiredep have looked for Module A in the directory specified by `.bowerrc`? 

This doesn't seem like an issue that targets grunt-wiredep, but more like an issue targetting wiredep itself.
